src/fu-tool.c

```diff
@@ -63,6 +63,16 @@ fu_sort_command_name_cb (FuUtilItem **item1, FuUtilItem **item2)
 	return g_strcmp0 ((*item1)->name, (*item2)->name);
 }
 
+static void
+fu_util_test_for_sandbox (const gchar *value, GError **error)
+{
+	g_autofree gchar *path = g_path_get_dirname (value);
+	if (!g_file_test (path, G_FILE_TEST_EXISTS | G_FILE_TEST_IS_DIR))
```

Comment:
you probably want braces here -- it's a bit lopsided. it' also needs to be called something like fu_util_perhaps_prefix_sandbox_error() as it's not actually testing for a sandbox, just making an error better -- better names welcome.

Reply:
fixed